composer.json

```diff
@@ -25,7 +25,8 @@
     },
     "require-dev": {
         "symfony/yaml": "~2.3|~3.0",
-        "phpunit/phpunit": "^5.4"
+        "phpunit/phpunit": "^5.4",
+        "malukenho/docheader": "^0.1.4"
```

Comment:
@malukenho one added dependency to just check some docblocks? I'm quite against this...